src/event_handler.cpp

```diff
@@ -139,29 +137,13 @@ void	EventHandler::HandlePollOutEvent(pollfd entry) {
 			}
 			//送信失敗
 			if (sent_msg_length < 0) {
-
-				switch (errno) {
-				//プログラム終了
-				case EFAULT:
-				case EMSGSIZE:
-					std::cout <<  strerror(errno) << std::endl;
-					std::exit(EXIT_FAILURE);
-				//次回POLLOUT発生時に再送
-				case EWOULDBLOCK:
-					return;
-				//直ちに再送
-			  case EINTR:
-					break ;
-				//接続切断
-				default:
-					Detach(entry);
-					Event event(entry.fd, entry.revents);
-					event.set_command(message::kQuit);
-					AddResponseMap(database_.ExecuteEvent(event));
-					response_map_.erase(target_fd);
-					return;
-				}
-				continue;
+				Detach(entry);
+				Event event(entry.fd, entry.revents);
+				Event* event_ptr = &event;;
+				event_ptr->set_command(message::kQuit);
+				ExecuteCommand(event_ptr);
+				response_map_.erase(target_fd);
+				return;
```

Comment:
ここは、エラーが発生したので強制QUITする、という場面ですよね？
であれば、checkは不要、、と言いたかったのですが、checkで情報を付加しないと何が起こるかわからなさそうなので、こちらの方(ExecuteCommandを呼び出す)が正しそうですね。
となると、以前の指摘はdatabase_.ExecuteEvent関数を呼び出す場合に沿った指摘だったので、もう一度修正をお願いしたいです。これなら、Eventはnewしたほうが良いです。
```suggestion
				Detach(entry);
				Event* event_ptr = new Event(entry.fd, entry.revents);
				event_ptr->set_command(message::kQuit);
				ExecuteCommand(event_ptr);
				delete event_ptr;
				response_map_.erase(target_fd);
				return;
```